### DISCUSSION TITLE
OpenTelemetry Plugins

### DISCUSSION THREAD

**Discussion by insanpraja** [sparckles/Robyn#725] [Ideas]
Has anyone did this with Robyn, it would makes Robyn super transparent on traces, latencies, and performances...

**Comment by sansyrox** [comment]
Hey @insanpraja 👋 

>  super transparent on traces, latencies, and performances...

Can you please explain what you mean by "transparency" here? 😅

**Reply by insanpraja** [reply]
Hi @sansyrox,
Basically I was using SigNoz and bootstrap my python backend apps with OpenTelemetry, and basically I can see almost everything, traces etc.. things that we see with flamegraph. That is it.

Thanks
